chain/network/src/network_protocol/proto_conv/peer_message.rs

```diff
@@ -265,9 +266,18 @@ impl TryFrom<&proto::PeerMessage> for PeerMessage {
             ProtoMT::Transaction(t) => PeerMessage::Transaction(
                 SignedTransaction::try_from_slice(&t.borsh).map_err(Self::Error::Transaction)?,
             ),
-            ProtoMT::Routed(r) => PeerMessage::Routed(Box::new(
-                RoutedMessage::try_from_slice(&r.borsh).map_err(Self::Error::Routed)?,
-            )),
+            ProtoMT::Routed(r) => PeerMessage::Routed(Box::new(RoutedMessageV2 {
+                msg: RoutedMessage::try_from_slice(&r.borsh).map_err(Self::Error::Routed)?,
+                created_at: r.created_at.as_ref().and_then(|timestamp| {
+                    match utc_from_proto(timestamp) {
+                        Ok(utc) => Some(utc),
+                        Err(err) => {
+                            tracing::warn!(target: "network", err=?err, "Malformed routed message created_at timestamp");
```

Comment:
nit: I'd return an error if timestamp is not parseable: it is allowed to be missing, but not incorrect.